pkg/sfu/streamtrackermanager.go

```diff
@@ -367,18 +401,6 @@ func (s *StreamTrackerManager) getLayeredBitrateLocked() ([]int32, Bitrates) {
 		}
 	}
 
-	if s.isSVC {
```

Comment:
use frame dependency to calculate bitrates of layer in streamtrack_dd

Reply:
We will need this if we ever use VP9 without DD. Change the check to `isSVC && ddTracker == nil`?